app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
```

Comment:
Suggest using absolute import of basic modules for better readability.
`sys.argv` more informative than simple `argv`, that can be simple named variable